regression/contracts/named-contracts/test-fail1.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main-fail1.c
+
+^main-fail1.c.* error: the contract for function 'foo' must be attached to the first declaration.$
```

Comment:
This error message sounds suspicious when it says "first declaration": there should not be a particular order of declarations required, unless this really is restricting to single translation units? What happens when linking declarations (and perhaps: definitions) from multiple translation units?